pkg/service/replay/replay.go

```diff
@@ -711,3 +713,84 @@ func (r *Replayer) ProvideMocks(ctx context.Context) error {
 	<-ctx.Done()
 	return nil
 }
+
+// Normalise initiates the normalise process for normalising the test cases.
+func (r *Replayer) Normalise(_ context.Context, cfg *config.Config) error {
+	path := cfg.Path
+	testSets := viper.GetString("test-sets")
```

Comment:
These are not needed use config struct instead of getting them like this. Please refer to how test or record command are accessing these kind of flags